portfolio/src/main/webapp/results.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Results</title>
+    <link rel="stylesheet" href="alexFStyle.css">
+    <script src="script.js"></script>
+  </head>
+  <body onload="getResult()">
+    <div class="main-header">
+        <nav class="nav main-nav">
+            <ul>
+                <li><a href="index.html">HOME</a></li>
+                <li><a href="history.html">HISTORY</a></li>
+            </ul>
+        </nav>
+      <h1 class="header">Results</h1>
+    </div>
+    <div class="content">
+      <p id="result-container" class="result"></p> 
+      <button onclick=unlikeSong()>I don't like this song.</button><br><br>
```

Comment:
Would we want another button for if they like the song?

Reply:
I think this could also be okay for now. later on we can add a like button. Personally, I don't usually "like" songs, I just unlike them so i dont hear them again so this makes sense to me.

Reply:
I agree, for now I think an unlike button is fine and later we can add it. I would say we should include a like button if we are going to have a page where users can see all their liked songs.